Source/Core/DolphinNoGUI/CMakeLists.txt

```diff
@@ -32,6 +33,13 @@ PRIVATE
   cpp-optparse
 )
 
+if(WIN32)
+  target_link_libraries(dolphin-nogui PRIVATE d3d11)
+	if(NOT MSVC)
+  	target_link_options(dolphin-nogui PRIVATE -municode)
+	endif()
+endif()
```

Comment:
Ditto regarding spaces over tabs. Tabs and spaces are mixed here.

Reply:
done